manifests/plugin.pp

```diff
@@ -22,8 +22,8 @@
   $config_content  = undef,
   $update_url      = undef,
   $plugin_dir      = '/var/lib/jenkins/plugins',
-  $username        = 'jenkins',
-  $group           = 'jenkins',
+  $username        = '',
+  $group           = '',
```

Comment:
I can't come up with a [good] reason why the plugins should be owned by a different user than the jenkins master. I feel we should probably depreciate / ignore these params and allow overriding the default user/group only via params on the `jenkins` class.